Dockerfile

```diff
@@ -1,9 +1,18 @@
-FROM node:10-buster
+FROM node:14.21.3-buster
 
-ARG build_branch=master
-                                        
-RUN git clone --depth=1 --recursive --shallow-submodules -b $build_branch https://github.com/ONLYOFFICE/document-server-integration.git
+ENV DOCS_LAN='documentserver'
```

Comment:
I don't see there we using it? I see calls to LAN, not DOCS_LAN

Reply:
Also maybe change a name var - it's not an LAN, I'm not quite sure that it is entity (maybe `network-name`) but for sure not LAN

Reply:
Is SERVICE or SARVICE_ALIAS better?

Those compose uses service names as an alias to the name on the network

https://docs.docker.com/compose/compose-file/05-services/#aliases

Reply:
This will not work, since the local ip is registered in the frontend for opening an iframe with an editor

Obviously, if you register a document server there, the editor will not start
```html
<script type="text/javascript" src="http://documentserver:8008/web-apps/apps/api/documents/api.js"></script>
```

Reply:
> Is SERVICE or SERVICE_ALIAS better?

DOCSERVER_NETWORK_NAME?

Reply:
> Also maybe change a name var - it's not an LAN, I'm not quite sure that it is entity (maybe `network-name`) but for sure not LAN

Maybe just `DOCS_ADDRESS`?


Or based on the definition of [RFC 1918](https://www.rfc-editor.org/rfc/rfc1918), call `PRIVATE_IP`